app/export.go

```
@@ -174,28 +176,28 @@
 	// iterate through unbonding delegations, reset creation height
-	app.StakingKeeper.IterateUnbondingDelegations(
-		ctx,
-		func(_ int64, ubd stakingtypes.UnbondingDelegation) (stop bool) {
-			for i := range ubd.Entries {
-				ubd.Entries[i].CreationHeight = 0
-			}
-			app.StakingKeeper.SetUnbondingDelegation(ctx, ubd)
-			return false
-		},
-	)
+	app.StakingKeeper.IterateUnbondingDelegations(ctx, func(_ int64, ubd stakingtypes.UnbondingDelegation) (stop bool) {
```

golangci-lint (1.62.2)

Error return value of `app.StakingKeeper.IterateUnbondingDelegations` is not checked

(errcheck)